docs/datasources/index.mdx

```diff
@@ -0,0 +1,30 @@
+---
+description: |
+  The Azure plugin is able to fetch data from Azure. To achieve this, the plugin comes with
+  data sources to retrieve keyvault secrets.
+page_title: Azure - Data Sources
+sidebar_title: Overview
```

Comment:
Sidebar_title works but is old, we are using nav_title now. 
```suggestion
nav_title: Overview
```

Reply:
Done